awscli/customizations/configure/get.py

```diff
@@ -34,8 +34,12 @@ class ConfigureGetCommand(BasicCommand):
          'cli_type_name': 'string', 'positional_arg': True},
     ]
 
-    def __init__(self, session, stream=sys.stdout, error_stream=sys.stderr):
+    def __init__(self, session, stream=None, error_stream=None):
         super(ConfigureGetCommand, self).__init__(session)
+        if stream is None:
+            stream = sys.stdout
+        if error_stream is None:
+            error_stream = sys.stderr
```

Comment:
This was updated because of the method default variable bug that every python programmer encounters from time to time. Because these defaults are evaluated before runtime, they are not properly mocked out by the test runner. Same issue for `list.py`.